tests/integ/test_auto_ml.py

```diff
@@ -49,6 +48,14 @@
 }
 
 
+# use a succeeded AutoML job to test describe and list candidates method, otherwise tests will run too long
+# reusable-job will be created once if it doesn't exist, and be reused in relevant tests.
+@pytest.fixture(scope="module")
+def reusable_job_name():
```

Comment:
This looks a little confusing name as `unique_name_from_base` will generate a new name every time, so is it really reusable?

Reply:
Right, I was trying to keep with the previous intention of `AUTO_ML_JOB_NAME = "python-sdk-integ-test-base-job"`. That job name was hard coded and used in some tests with the `.create_auto_ml_job_if_not_exist()` function. This would end up creating the job once for the lifetime of the account and reused indefinitely. This is problematic since it created parity in accounts that created this job before and after intelligent defaults. 

The `reusable_job_name()` instead will result in creating a job once per test session with `.create_auto_ml_job_if_not_exist()` and reuse it where possible. Maybe can change the name to `test_session_job_name()`?

Reply:
changed to `test_session_job_name()`